fitters/src/DAF.cc

```diff
@@ -214,6 +234,29 @@ void DAF::setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps)
   }*/
 }
 
+void DAF::setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps, unsigned int minIter, unsigned int maxIter) {
+  // The betas are calculated as a geometric series that takes nSteps
+  // steps to go from bStart to bFinal.
+  assert(bStart > bFinal);
+  assert(bFinal > 1.E-10);
+  assert(nSteps > 1);
+
+  minIterations_ = minIter;
+  maxIterations_ = maxIter;
+
+  betas_.clear();
+
+  for (unsigned int i=0; i<nSteps; ++i) {
```

Comment:
If you want to be fancy...

```
double ratio = pow(bFinal / bStart, 1.0 / (nSteps - 1.0));
for (unsigned int i=0; i<nSteps; ++i) {
  betas_.push_back(bStart * pow(ratio, i));
}
```

See https://godbolt.org/z/PxebYsYY7 .

Reply:
This is copy-pasted from the already present setAnnealingScheme. I would like to keep identical, but of real substantial changes, to help a reader in understand the differences of the two.